Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM nginx:alpine
+COPY ./build /usr/share/nginx/html
```

Comment:
the build output is in `build` folder, so I had to update it.

Reply:
OK :+1: